### PR TITLE
Show user name instead of email and make dropdown menu right-aligned

### PR DESCRIPTION
**Pull request by jameshowardwang**
# Description

We should make users identify with their name on the site, not their email, and the menu shouldn't be clipped on the right for short names. Tested in Chrome, Safari, and Firefox.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix a feature that knowingly causes existing functionality to not work as expected)

# Test and Reproduce
Outline the steps to test or reproduce the PR here.

1. 

## Impacted Areas in Application
List general components of the application that this PR will affect:

Which pages?

- [ ] Login Screen
- [ ] Single Upload Page
- [ ] Batch Upload Page
- [ ] All Projects Page
- [ ] Sample Details Page
- [ ] Report Page

Specific components 

* 

# Testing Script:

###Login Page
* Successfully login to the application and get taken to the project page

###Single Upload
* Using an AWS path that doesn't exist throws an error
* Succesfully uploaded files from a folder (use test host)
* Uploading a duplicate file throws an error
* Pipeline gets initiated successfully 

###Batch Upload
* Using an AWS path that doesn't exist throws an error
* Succesfully uploaded files from a folder (use test host)
* Navigate to batch upload settings page after upload 
* Host selection from previous page still selected 
* Uploading a duplicate file throws a warning
* Pipeline gets initiated successfully 

###All Project Page
* Switch between projects 
* Search for a sample
* Navigate through different pages of samples
* Select Upload new sample button
* Select sample to see generated report

###Sample Details Page
* Information is consistent from Project Page
* Notes are editable and save successfully 
* Can navigate to report page 

###Report Page
* Data loads succesfully 
* Collapse and Uncollapse button works
* Clicking on the taxonomy name gives you total reads for that taxon
* Category filter works
* Genus Search works
* Filters work as expected
* Disabling Filters works
* Filtering by column works 

# Checklist:

- [x] I have run through the testing script to make sure current functionality is unchanged
- [x] I have done relevant tests that prove my fix is effective or that my feature works
- [x] I have spent time testing out edge cases for my feature
- [x] I have updated the test script or pull request template if necessary
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/assets/src/styles/_header.scss [hunk 1]
```diff
@@ -142,18 +126,22 @@ a {
...
+  float: right;
   margin-top: 1rem;
   margin-right: 0.5rem;
+  background-color: rgba(0, 0, 0, 0);
+  min-height: 0;
+  box-shadow: none;
+  border: none;
   .text {
-    font-size: 1rem !important;
+    font-family: Open Sans, Helvetica Neue, Arial, Helvetica, sans-serif;
```
Comment (jshoe) [inline app/assets/src/styles/_header.scss]: for font-family could you import themes/_typograpy and then use $font-family-sans-serif

File: app/assets/src/styles/_header.scss [hunk 2]
```diff
@@ -142,18 +126,22 @@ a {
   }
 }
 
-.profile-header-dropdown {
-  border: none !important;
+.profile-header-menu.menu {
+  float: right;
   margin-top: 1rem;
   margin-right: 0.5rem;
+  background-color: rgba(0, 0, 0, 0);
```
[the remark this answers is not in the document]
Comment (jameshowardwang) [reply app/assets/src/styles/_header.scss]: i wanted it to be transparent (so 0 alpha) - do we have a color for that?

File: app/assets/src/styles/_header.scss (hunk 1 above)
Comment (jameshowardwang) [reply app/assets/src/styles/_header.scss]: whoops i committed before seeing this - will clean up in follow-up diff

Comment (jameshowardwang) [reply app/assets/src/styles/_header.scss]: nm, i see i can do rgba($black, 0)

File: app/assets/src/styles/_header.scss (hunk 2 above)
[the remark this answers is not in the document]
Comment (tfrcarvalho) [reply app/assets/src/styles/_header.scss]: css' `transparent` as in `background-color: transparent` :)
